managedidentity.tf

```
@@ -11,7 +11,7 @@ resource "azurerm_role_assignment" "kvaccess" {
   scope                = azurerm_key_vault.kv.id
 }
 resource "azurerm_role_assignment" "kvgroupaccess" {
-  principal_id         = data.azuread_group.group.object_id
+  principal_id         = var.platform_operations_group
```

should be what is the requirement and not the implementation
```suggestion
  principal_id         = var.administrator_ad_group_object_id
```